mmcls/models/backbones/gcvit.py

```diff
@@ -0,0 +1,792 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# by lihao
+# Reproduction for GCViT
+
+import torch
+import torch.nn as nn
+from mmengine.model.weight_init import trunc_normal_
+
+from mmcls.registry import MODELS
+
+
+def drop_path(x,
+              drop_prob: float = 0.,
+              training: bool = False,
+              scale_by_keep: bool = True):
+
+    if drop_prob == 0. or not training:
+        return x
+    keep_prob = 1 - drop_prob
+    shape = (x.shape[0], ) + (1, ) * (
+        x.ndim - 1)  # work with diff dim tensors, not just 2D ConvNets
+    random_tensor = x.new_empty(shape).bernoulli_(keep_prob)
+    if keep_prob > 0.0 and scale_by_keep:
+        random_tensor.div_(keep_prob)
+    return x * random_tensor
+
+
+class DropPath(nn.Module):
```

Comment:
Please use the DropPath of mmcv to replace this module, if the implementation is equal.

```python
from mmcv.cnn.bricks import DropPath
```